File: frontend-android/clases/Reto.kt
package com.example.myapplication

data class Reto(
    var retoId: Int,
    var nombre: String,
    var descripcion: String,
    var puntos: Int
) {
    constructor() : this(
        retoId = 0,
        nombre = "",
        descripcion = "",
        puntos = 0
    )
}

File: frontend-wpf/AplicacionEscritorioWPF/PrototipadoEscritorio/PrototipadoEscritorio/Models/Reto.cs
using CommunityToolkit.Mvvm.ComponentModel;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using static System.Runtime.InteropServices.JavaScript.JSType;

namespace PrototipadoEscritorio.Models
{
    public partial class Reto : ObservableObject
    {
        public int RetoId { get; set; } // No reactivo

        [ObservableProperty]
        private string nombre;

        [ObservableProperty]
        private string descripcion;

        [ObservableProperty]
        private int puntos;

        public Reto() { }

        public Reto(int retoId, string nombre, string descripcion, int puntos)
        {
            RetoId = retoId;
            Nombre = nombre;
            Descripcion = descripcion;
            Puntos = puntos;
        }
    }
}

